tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5918,20 +5987,160 @@ <h2>Show similarity to selected datapoint</h2>
                 const attribSparseKeyKey =
                   attributionPrefix + keys[j] + attributionSparseKeySuffix;
                 datapoint[attribSparseKeyKey] = attribs[keys[j]][0];
-                const attribSparseValueKey =
+                attribKey =
```

Comment:
The value key is used down below now to find the global max and min attributions, so in both the case of normal or sparse attributions, we need to be able to key into the attribution value for bounds checking. Since the sparse key isn't needed for that part, didn't need to change its name